### PR TITLE
Ft user states

### PR DESCRIPTION
**Pull request by FelixWambiri**
**Description of Task to be completed?**
*  To list only active members in the default list and have a second overview page for the deactivated ones. All users list should show active and inactive users without navigating to the active and inactive pages.

**How should this be manually tested?**
* Login as an admin with password admin
* Create a new gym and give it a name of your choice
* Add a few members to the gym.
* As an administrator use your permissions to deactivate a few users. 
* Go back to gym overview and click on the all, active and deactivated buttons to view all users.

**What are the relevant pivotal tracker stories?**
* #157507861

**Background context**
* At the moment, the only way to determine whether a gym member is active or not is to open the relevant pages, that is the active and inactive user pages.

**Screenshots**
* Default page which also happens to show all users and their states.

image
* Page showing Active users after the active users button is clicked

image
* Page showing inactive users after clicking the deactivated users button.

image

### CODE REVIEW

File: wger/gym/templates/gym/partial_user_list.html [hunk 1]
```diff
@@ -37,6 +37,16 @@
     <td data-order="{{current_user.last_log|date:'U'}}">
         {{current_user.last_log|default:'-/-'}}
     </td>
+    <!-- Add colum to show if the user is active or not -->
```
Comment (meshnesh) [inline wger/gym/templates/gym/partial_user_list.html:40]: Good choice using icons to show active and inactive uses.

**Review by meshnesh — APPROVED** [review]
Nice implementation of your task and finishing it.

**Review by mutoromike — APPROVED** [review]
LGTM

**Review by betsybeth — APPROVED** [review]
LGTM

**Review by Kalela — APPROVED** [review]
LGTM

**Review by Kalela — APPROVED** [review]
Great job implementing feedback. LGTM

**Review by frankip — APPROVED** [review]
I like the use of the icons to show the activated and deactivated users
👍

**Pull request merged**